React/Base/RCTUIKit.h

```diff
@@ -449,10 +449,6 @@ NS_INLINE BOOL RCTUIViewIsDescendantOfView(RCTPlatformView *view, RCTPlatformVie
   return [view isDescendantOf:parent];
 }
 
-NSArray *RCTUIViewCalculateKeyViewLoop(RCTPlatformView *root);
```

Comment:
Why are we deleting these? Can a client not still use these if they don't care to set any custom key-view looping?